features/bevragen/zoek-met-postcode-en-huisnummer/fout-cases.feature

```diff
@@ -190,12 +103,14 @@ Rule: Postcode en huisnummer zijn verplichte parameters
     | required | <foutieve parameter> | Parameter is verplicht. |
 
     Voorbeelden:
-    | titel                             | postcode | huisnummer | foutieve parameter |
-    | Zoek met alleen valide huisnummer |          | 2          | postcode           |
-    | Zoek met alleen valide postcode   | 2628HJ   |            | huisnummer         |
+    | titel                                              | postcode | huisnummer | foutieve parameter |
+    | Een lege string is opgegeven als huisnummer waarde |          | 2          | postcode           |
```

Comment:
de titel klopt niet bij de testsituatie, hier is postcode leeg. andersom bij volgende voorbeeld

Reply:
eventueel titels omwisselen.